pkg/sessionctx/variable/session.go

```diff
@@ -2391,15 +2431,6 @@ func (s *SessionVars) SetStringUserVar(name string, strVal string, collation str
 	}
 }
 
-// UnsetUserVar unset an user defined variable by name.
-func (s *SessionVars) UnsetUserVar(varName string) {
```

Comment:
`UnsetUserVar` has been moved to: https://github.com/pingcap/tidb/pull/55866/files#diff-1df3757be8063f9a9a3c70650f58368402d93bbf842ac2e1c2f8426542f1a2f5R743